src/register/RegistrationPage.jsx

```diff
@@ -86,6 +87,7 @@ const RegistrationPage = (props) => {
   const { formatMessage } = useIntl();
   const countryList = useMemo(() => getCountryList(getLocale()), []);
   const queryParams = useMemo(() => getAllPossibleQueryParams(), []);
+  const { variant } = queryParams;
```

Comment:
Similar approach can be used as above. 
```suggestion
  const isRegistrationEmbedded = queryParams?.variant === 'embedded';
```